### DISCUSSION TITLE
inbound erc20 transfers

### DISCUSSION THREAD

**Discussion by univerz** [otterscan/otterscan#88] [Q&A] [answered]
does otterscan handle inbound erc20 tranfers in address transactions listing?

**Comment by wmitsuda** [comment, marked as answer]
Not yet!

Technically speaking, the transactions listing displays only tx where the address state was "touched", and ERC20 transfers don't actually "touch" an address, but the state of the ERC20 contract itself.

But this is an expected feature we want to have, I'll leave this open to track it and others comment too.

**Comment by wmitsuda** [comment]
BTW, the v2.0.0 release finally have this capability!

See: https://github.com/otterscan/otterscan/releases/tag/v2.0.0
